test/test-applications/integrations/LogsInjection.Serilog/LogsInjection.Serilog.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">2.0.0</ApiVersion>
+    <DefineConstants Condition="'$(ApiVersion)'&gt;='2.0.0'">$(DefineConstants);SERILOG_2_0</DefineConstants>
+    <DefineConstants Condition="'$(ApiVersion)'&gt;='1.4.152' and '$(ApiVersion)'&lt;'2.0.0'">$(DefineConstants);SERILOG_LOG_OPTIN_CROSS_APPDOMAIN</DefineConstants>
+    <DefineConstants Condition="'$(ApiVersion)'&lt;'1.4.152'">$(DefineConstants);SERILOG_CROSS_APPDOMAIN_FAILURE</DefineConstants>
+
+    <!-- Required to build multiple projects with the same Configuration|Platform -->
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+
+    <ExcludeManagedProfiler>true</ExcludeManagedProfiler>
+    <ExcludeNativeProfiler>true</ExcludeNativeProfiler>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Serilog" Version="$(ApiVersion)" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(ApiVersion)' &gt;= '2.0.0'">
+    <!-- Use somewhat older versions of Serilog packages for easiest compatibility -->
+    <PackageReference Include="Serilog.Formatting.Compact" Version="1.0.0" />
+    <PackageReference Include="Serilog.Sinks.File" Version="2.2.0" />
+    
+    <!-- Directly reference dependencies of Serilog.Sinks.File to avoid package downgrade errors -->
+    <PackageReference Include="System.IO" Version="4.3.0" />
+    <PackageReference Include="System.Text.Encoding.Extensions" Version="4.3.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(ApiVersion)' &gt;= '2.0.0' and '$(TargetFramework)' != 'net452'">
+    <!-- Directly reference dependencies of Serilog.Sinks.File to avoid package downgrade errors -->
+    <PackageReference Include="System.IO.FileSystem" Version="4.3.0" />
+    <PackageReference Include="System.IO.FileSystem.Primitives" Version="4.3.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\dependency-libs\LogsInjectionHelper\LogsInjectionHelper.csproj" />
+    <ProjectReference Include="..\dependency-libs\PluginApplication\PluginApplication.csproj" />
+    <ProjectReference Include="..\..\..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
```

Comment:
Do we need/want this reference? It's currently referenced transitively, and isn't used directly by this app?

Reply:
Fixed in a1c64bfee75f3e355c2262dc33fc5dab3846e54e